docs/documentation/index.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: page
+permalink: /documentation/
+toc: true
+
+# Banner Content
+# =================================
+title: DSA/HistomicsTK Documentation
```

Comment:
Because of the font we use, I think this would scan better with spaces around the `/` (i.e., `DSA / HistomicsTK`).